### ISSUE TITLE
Display error messages better

### ISSUE THREAD

**Issue by oakkitten** [ubergeek42/weechat-android#587]
At the moment, we use Toasts for error messages. This was “good enough” before, but as these are limited to just two lines on recent Android versions, we should switch to Snackbars. A snackbar could have a button to show the full traceback and whatnot.

**Comment by freedom-foundation** [comment]
No. You are lessening my [bug] #583. Downplaying critical security bug into pettiness. I feel like I am getting "trolled" and you should know it work's.

**Comment by oakkitten** [comment]
While we are at it, it would probably help to have r8 not combine several exceptions into one, it's just very confusing
